docs/pathways/ontology-engineer.md

```diff
@@ -45,17 +44,11 @@ This section is a non-ordered collection of how to documents that an engineer mi
 
 - [Adding taxon restrictions](../howto/add-taxon-restrictions.md)
 - [Changing files in pull requests](../howto/change-files-pull-request.md)
-- [Cloning a repo](../howto/clone-mondo-repo.md)
```

Comment:
For the other ones I removed from the Ontology Engineer "how to" section, it's either because (a) it is very basic, e.g. cloning, and a quick google search away if really needed, or (b) it's already in another section of the same page.